turbot/__init__.py

```diff
@@ -110,18 +110,9 @@ def discord_user_id(channel, name):
     return getattr(discord_user_from_name(channel, name), "id", None)
 
 
-def is_turbot_admin(channel, user):
-    """Checks to see if user has the Turbot Admin role in this server"""
-
-    # Is there a way to go from User to Member to roles?
-    # member.roles
-    member = channel.guild.get_member(user.id)
-    if member:  # Since the user sent a message, they should be a member
-        for role in member.roles:
-            if str(role) == "Turbot Admin":
-                return True
-
-    return False
+def is_turbot_admin(user):
+    """Checks to see if user has the Turbot Admin role in this server."""
+    return any(role.name == "Turbot Admin" for role in user.roles)
```

Comment:
Indeed, `member.roles` exists and seems to work 🎉

Reply:
Ah, there's a bit of a snafu here that we need to be careful about.  The author field in a message is a Member (class that actually has 'roles') most of the time, but it CAN return a User (If the user left the server or sent a private message) which doesn't have the 'roles' member inside.

While it's unlikely we'll hit those User cases today, we may want to extend to supporting private messaging in the future (In which case the role-based permissions won't work anyway, but I digress).  I think we should either continue to assume that the author is a User, or do a type-check to see if it's a Member or User.

Reply:
Oh Python, you so silly.

Reply:
Updated PR to handle both user or member.